tools/sof-kernel-log-check.sh

```diff
@@ -268,6 +268,11 @@ case "$platform" in
         # Bug Report: https://github.com/thesofproject/sof-test/issues/838
         # New TGLU_UP_HDA_ZEPHYR device reporting "TPM interrupt not working" errors.
         ignore_str="$ignore_str"'|kernel: tpm tpm0: \[Firmware Bug\]: TPM interrupt not working, polling instead'
+	# 'failed to change power setting' and other errors observed at
+	# boot time on one TGLU_VOLT_SDW. Internal issue #174.  GOODIX
+	# touchscreen
+	# /sys/devices/pci0000:00/0000:00:15.1/i2c_designware.1/i2c-1/i2c-GDIX0000:00
+	ignore_str="$ignore_str"'|kernel: i2c_hid_acpi i2c-GDIX0000:00'
```

Comment:
It's not your fault, but someone must have used a windows program to edit this file previously, could you de-indent line 267?  Don't worry, I checked git blame and spoke sternly in the mirror.